packages/neptune-extension/src/common/hooks/activation.ts

```diff
@@ -22,6 +30,16 @@ export function createActivationHandler(platformNotebook: PlatformNotebook) {
     notebook,
   } = useSelector(getNotebookState);
 
+  let notebookDeps: NotebookDependencies = [undefined, undefined, undefined, undefined];
+
+  if (notebook) {
+    notebookDeps = [notebook.id, notebook.path, notebook.organizationName, notebook.projectName]
```

Comment:
orgName and projName will be added by my other PR: https://github.com/neptune-ml/neptune-notebooks/pull/111